CONTRIBUTING.md

```diff
@@ -216,24 +220,23 @@ We use `pip` to install other Python packages. You may need to install `pip` if
 To determine if you need to, type the following command into the terminal:
 
 ```bash
-type pip
```

Comment:
There is a ```type``` command on Windows, it does basically what ```less``` does on *nix I guess. So not what these instructions expect ;) Rather than writing two sections I figured out trying to call it with --version would be reasonable and cross-platform.